test/image/mocks/funnel_axis_with_other_traces.json

```diff
@@ -0,0 +1,196 @@
+            "yaxis": "y"
+        },
+        {
+            "type": "scatter",
+            "orientation": "v",
+            "x": [
+                "A",
+                "B",
+                "C"
+            ],
+            "y": [
+                5,
+                4,
+                3
+            ],
+            "xaxis": "x",
+            "yaxis": "y"
+        },
+        {
+            "type": "funnel",
+            "width": 0.6,
+            "opacity": 0.8,
+            "orientation": "v",
+            "x": [
+                "A",
+                "B",
+                "C"
+            ],
+            "y": [
+                5,
+                4,
+                3
+            ],
+            "xaxis": "x2",
+            "yaxis": "y2"
+        },
+        {
+            "type": "bar",
+            "width": 0.4,
+            "orientation": "v",
+            "x": [
+                "A",
+                "B",
+                "C"
+            ],
+            "y": [
+                5,
+                4,
+                3
+            ],
+            "xaxis": "x2",
+            "yaxis": "y2"
+        },
+        {
+            "type": "funnel",
+            "width": 0.6,
+            "opacity": 0.8,
+            "orientation": "h",
+            "x": [
+                5,
+                4,
+                3
+            ],
+            "y": [
+                "A",
+                "B",
+                "C"
+            ],
+            "xaxis": "x3",
+            "yaxis": "y3"
+        },
+        {
+            "type": "waterfall",
+            "width": 0.4,
+            "orientation": "h",
+            "x": [
+                3,
+                -2,
+                0
+            ],
+            "y": [
+                "A",
+                "B",
+                "C"
+            ],
+            "measure": [
+                "r",
+                "r",
+                "t"
+            ],
+            "xaxis": "x3",
+            "yaxis": "y3"
+        },
+        {
+            "type": "funnel",
+            "width": 0.6,
+            "opacity": 0.8,
+            "orientation": "h",
+            "x": [
+                5,
+                4,
+                3
+            ],
+            "y": [
+                "A",
+                "B",
+                "C"
+            ],
+            "xaxis": "x4",
+            "yaxis": "y4"
+        }
+    ],
+    "layout": {
+        "width": 800,
+        "height": 800,
+        "dragmode": "pan",
+        "xaxis": {
+            "type": "category",
+            "domain": [
+                0,
+                0.48
+            ],
+            "autorange": true
+        },
+        "xaxis2": {
+            "type": "category",
+            "domain": [
+                0.52,
+                1
+            ],
+            "autorange": true
+        },
+        "xaxis3": {
+            "domain": [
+                0,
+                0.48
+            ],
+            "autorange": true
```

Comment:
... and no need to set `autorange: true` for all these axes. It is the default.

Reply:
Thanks for the note. Done in https://github.com/plotly/plotly.js/pull/3817/commits/020a34b9744acb754eb41ee101ff8e4788cf96d9.

Reply:
Thanks!